Automattic/jetpack
# AI Featured Image: Handling errors

Add an initial interface to handling errors when generating an image.

## Proposed changes:

* Add `reject` calls in generate image hook.
* Set error in state in featured image.
* Use error in state to show a feedback for the user and try again option.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Make sure you have the beta blocks enabled.
* Sandbox the public-api and fail the request on purpose.
* You should see the error interface with try again button.

#### Demo 💻 

![image](https://github.com/Automattic/jetpack/assets/1663717/518ee84e-1d1f-453a-ac72-f2166d180d39)
